proto/LogProtocol.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+option optimize_for = SPEED;
+
+enum ColumnFlagType {
+    BinaryFlag = 0;
+}
+
+message TableName {
+	string schema = 1;
+	string table = 2;
+	int64 tableID = 3;
+	int64 partition = 4;
+}
+
+message Column {
+	bytes type = 1;
+	bool whereHandle = 2;
+	oneof columnType_present{
+    	ColumnFlagType flag= 3;
+    }
```

Comment:
We use a uint64 to store the column flag, and it can contain multiple ColumnFlag via `or` operation.
Besides the `whereHandle` can be removed now.